install/v0.12.0-rc1/kserve-cluster-resources.yaml

```diff
@@ -338,6 +338,40 @@ spec:
     version: "1"
 ---
 apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
```

Comment:
This folder is for `v0.12.0-rc1` which is already released. You can remove this

Reply:
What folder can I add changes to installation files?

Reply:
It will be done as part of the release process